sda-download/dev_utils/make_certs.sh

```diff
@@ -26,4 +30,5 @@ openssl x509 -req -in ./certs/client.csr -days 1200 -CA ./certs/ca.pem -CAkey ./
 openssl req -config "$(dirname "$0")"/ssl.cnf -new -nodes -newkey rsa:4096 -keyout ./certs/download-key.pem -out ./certs/download.csr -extensions download_cert
 openssl x509 -req -in ./certs/download.csr -days 1200 -CA ./certs/ca.pem -CAkey ./certs/ca-key.pem -set_serial 01 -out ./certs/download.pem -extensions download_cert -extfile "$(dirname "$0")"/ssl.cnf
 
+cp ./certs/ca.pem ./certs/ca-certificates.crt
```

Comment:
Is this line really needed?

Reply:
No, most of this file can actually be removed.